lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -586,6 +586,12 @@ void OpenCLFunction::executeNCHWConvolution(
   addStringOption(options, "v_pad_A", "0");
   addStringOption(options, "v_pad_B", "0");
 
+  if (CC->getFusedActivation() == FusedActivation::RELU) {
+    addIntOption(options, "v_fuse_relu", 1);
+  } else {
+    addIntOption(options, "v_fuse_relu", 0);
```

Comment:
nit: `/* fuseReLU */ 0`

Reply:
Nor this. Note that these are specialization macro parameters for the other convolution.